src/c++/perf_analyzer/perf_analyzer.cc

```diff
@@ -52,6 +52,44 @@ SignalHandler(int signum)
     exit(0);
   }
 }
+
+bool
+IsLLMModel(
+    const std::shared_ptr<pa::ModelParser>& parser,
+    const pa::PAParamsPTR& params)
+{
+  bool is_llm_from_user = params->is_llm if (is_llm_from_user)
```

Comment:
this is valid c++ syntax?
i have only seen something like this construction in python or rust

Reply:
addressed in another comment 
👍🏼

Reply:
I think I'm missing a semicolon here.